Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -52,14 +70,30 @@ public ScanAPIModelWizardPage()
             xApisSelectionGrid.MarkUnMarkAllActive += MarkUnMarkAllActions;
             xApisSelectionGrid.btnRefresh.Visibility = Visibility.Visible;
             xApisSelectionGrid.btnRefresh.AddHandler(Button.ClickEvent, new RoutedEventHandler(BtnRefreshClicked));
+            xCompreExistingItemBtn.Visibility = Visibility.Collapsed;
             SetFieldsGrid();
         }
+
         private void BtnRefreshClicked(object sender, RoutedEventArgs e)
         {
             AddAPIModelWizard.IsParsingWasDone = false;
             Parse();
         }
 
+        void BtnCompareAPIClicked(object sender, RoutedEventArgs e)
+        {
+            AddAPIModelWizard.DeltaModelsList = new ObservableList<DeltaAPIModel>();
```

Comment:
more readable code should be:
AddAPIModelWizard.DeltaModelsList = APIDeltaUtils.ComparisonUtility(AddAPIModelWizard.LearnedAPIModelsList); (it will return list of Delta as output)
AddAPIModelWizard.DeltaModelsList =AddAPIModelWizard.DeltaModelsList;
 xApisSelectionGrid.InitViewItems();
xApisSelectionGrid.btnMarkAll.Visibility = Visibility.Collapsed;
xCompreExistingItemBtn.Visibility = Visibility.Collapsed;